### ISSUE TITLE
Bug: Index out of range exception in home

### ISSUE THREAD

**Issue by DaRealDingbat** [FluentHub/FluentHub#334]
### ⏱️ Before you start...

- [X] Have you updated FluentHub? You might be using an old version.
- [X] Have you checked if a similar issue has already been reported?

### 🪟 What version of Windows is this issue present?

Windows 11 Dev

### 🔢 What version/architecture of FluentHub are you on?

0.3.8.0 | X64 | RELEASE

### 📄 Description

Gives Error saying the index was out of range

### 🪜 Steps To Reproduce

1. Install The App For The First Time
2. Go To New Tav

### 🤔 Expected behavior

A home page.

### 📟 Stack Trace

<details><summary>Stack trace details</summary><div>

```
at System.ThrowHelper.ThrowArgumentOutOfRange_IndexException() + 0x16
   at FluentHub.Octokit.Queries.Users.NotificationQueries.<GetAllAsync>d__0.MoveNext() + 0xc95
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17
   at FluentHub.Uwp.ViewModels.Home.UserHomeViewModel.<LoadHomeContentsAsync>d__25.MoveNext() + 0x36b
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult() + 0xb
   at FluentHub.Uwp.ViewModels.Home.UserHomeViewModel.<LoadUserHomePageAsync>d__24.MoveNext() + 0x1e0
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult() + 0xb
   at Microsoft.Toolkit.Mvvm.ComponentModel.ObservableObject.<>c__DisplayClass20_0`1.<<SetPropertyAndNotifyOnCompletion>g__MonitorTask|0>d.MoveNext() + 0xab

```

</div></details>

### 📸 Assets

![image](https://user-images.githubusercontent.com/108444857/199640066-a31c5b96-b02c-4585-912f-4a688b892809.png)

**Comment by onein528** [comment]
This problem was already fixed in v0.5.0. The app which you in use is ild version and so you can install new one from GitHub Releases page.

Note: Because of the problem that we cannot generate the package for Microsoft Store, we decided we suspend to upload them for a while. Until it available in Store, you have no choice but downloading from GitHub Releases page. Sorry for any inconvenience and thank you for trying our app out.

**Comment by onein528** [comment]
This was fixed in v0.5
Can you try it again?
